lib/rails_critical_css_server/rewrite.rb

```diff
@@ -6,9 +6,8 @@ class Rewrite
 
     delegate :request, :params, to: :template
 
-    def initialize(template, manifest: template.asset_url('application.css'), version: Config.version, skip: false, cache: true, &block)
```

Comment:
The generated hash will be added to the filename, since the `asset_url` helper is smart enough to generate the correct digest and everything in a production environment.

This approach will also break because you may have more than one stylesheet in your HTML; for instance in one of my apps I have to split my css files up to avoid IE9 rule limit restrictions. So I have something like this:

    = RailsCriticalCssServer::Rewrite.call(self,
      manifest: asset_url('css_manifest.css')) do
      = stylesheet_link_tag 'bootstrap'
      = stylesheet_link_tag 'common'
      = stylesheet_link_tag 'page_rules'

`css_manifest.css` in this case is a single stylesheet with all the rules, in an easy format that criticalcss can read.